test/v2/config/environment/testnet.json

```diff
@@ -8,9 +8,10 @@
   ],
   "BLSOperatorStateRetrieverAddr":          "0x003497Dd77E5B73C40e8aCbB562C8bb0410320E7",
   "EigenDAServiceManagerAddr":              "0xD4A7E1Bd8015057293f0D0A557088c286942e84b",
-  "EigenDACertVerifierAddressQuorums0_1":   "0xFe52fE1940858DCb6e12153E2104aD0fDFbE1162",
+  "EigenDACertVerifierAddressQuorums0_1":   "0xd305aeBcdEc21D00fDF8796CE37d0e74836a6B6e",
```

Comment:
what is this new address?

Reply:
New cert verifiers I deployed for testing